WordPress/Classes/Models/Notifications/NotificationBlock.swift

```diff
@@ -68,6 +68,18 @@ class NotificationBlock: Equatable {
         type    = dictionary[BlockKeys.RawType] as? String
         text    = dictionary[BlockKeys.Text] as? String
     }
+
+    /// Forgive Me:
+    /// AVOID USING This Initializer at all costs. Consider getting the backend patched before creating these entities, locally.
+    ///
+    init(text: String?, ranges: [NotificationRange] = [], media: [NotificationMedia] = []) {
```

Comment:
For both this and the `NotificationRange` initializer, could you explain _why_ they shouldn't be used? (and perhaps in what situation they _should_ be used)